warehouse/models/intermediate/gtfs/int_gtfs_schedule__stop_times_grouped.sql

```diff
@@ -59,6 +66,7 @@ grouped AS (
         trip_end_timezone,
         iteration_num,
         exact_times,
+        MAX(_feed_valid_from) AS _feed_valid_from,
```

Comment:
Is there a specific reason you think it's preferable to group this as opposed to including it as part of the grouping? I probably would have just added it to the grouping logic; it won't change the grain and it will be unique with `feed_key` (like, a given `feed_key` necessarily comes with one and only one non-null `_feed_valid_from`)

Reply:
Fixed in [fb59602](https://github.com/cal-itp/data-infra/pull/2711/commits/fb596023e874459853c5f2b6bfdef501267cdd0c) - I wrote that before fully understanding the function of `_feed_valid_from`.